lib/iris/tests/test_basic_maths.py

```diff
@@ -494,15 +494,6 @@ def test_multiplication_not_in_place(self):
         b = iris.analysis.maths.multiply(a, 5, in_place=False)
         self.assertIsNot(a, b)
 
-    def test_type_error(self):
```

Comment:
Oh right. I was gonna make a comment. Personally, I don't think they're necessary. They're just saying 'don't pass the wrong type of argument to this function'. Well, you could have tests like that for any function, and that would force your to do instance checking at the top of every function. While we do instance check for the first argument to the function to insure it's a cube, we don't do it for the second argument, and the only reason these tests passed was because somewhere down the line it happened to raise a `TypeError`. My changes mean it now raises an `ArithmeticError`.

If you really want I'll add instance checks to make sure the tests pass, but I'm pretty against that idea. I suppose I would be willing to reinstate the first test, since we are doing the check for the first argument.